website/docs/topics/the-anatomy-of-unleash.mdx

```diff
@@ -0,0 +1,195 @@
+
+Each project must always have **at least one** active environment.
+
+Enterprise users can create and remove environments. Open-source and Pro customers get access to two environments: **development** and **production.**
+
+Environments are adjacent to [feature toggles](../reference/feature-toggles.mdx) in Unleash: neither one contains the other, but they come together to let you define activation strategies.
+
+
+
+<Figure img="/img/anatomy-of-unleash-new-feature-rollout.png" caption="You can use different activation strategies and constraints in different environments. For instance, you can show a feature only to select user IDs in development, but roll it out to 25% of your user base in production."/>
+
+## Features (feature  toggles)
+
+[**Feature toggles**](../reference/feature-toggles.mdx) are at the heart of Unleash’s functionality. Feature toggles belong to projects and live next to project environments. In and of itself, a feature toggle doesn’t do anything. You must assign [**activation strategies**](../user_guide/activation-strategies.md) to it for it to start taking effect.
+
+When creating a feature toggle, you must assign a unique (across your Unleash instance) name, a [feature toggle type](../advanced/feature-toggle-types.md), a [project](../user_guide/projects.md) it belongs to, and an optional description. Everything except for the name can be changed later.
+
+<Figure img="/img/anatomy-of-unleash-features.png" caption="Feature toggle states are evaluated independently in each environment." alt="A hierarchy showing a project containing an environment containing a feature toggle configuration."/>
+
+## Activation strategies
+
+<Figure img="/img/anatomy-of-unleash-strategy.png" caption="Activation strategies are applied to feature toggles on a per-environment basis and decide whether a feature is enabled or not." alt="A hierarchy displaying an environment containing a feature toggle configuration with an activation strategy."/>
+
+[**Activation strategies**](../user_guide/activation-strategies.md) (or just **strategies** for short) are the part of feature toggles that tell Unleash **who should get a feature**. An activation strategy is assigned to **one **feature toggle in **one **environment.
+
+When you check a [feature toggle](../reference/feature-toggles.mdx) in an application, the toggle’s strategies decide the result. As long as **at least one** of a toggle’s strategies resolve to true for the current context (user or application), then the toggle will be considered enabled. In other words, if you have a hundred strategies and ninety-nine of them resolve to false, but one of them resolves to true, then the toggle is enabled.
```

Comment:
I think this is probably clear enough and you do get a warning in the UI but if your environment is disabled this will short circuit in the SDKs to false

It might be worth pointing out that the environment to check in the application is decided by your API key

Reply:
These are good points, for sure.

Reply:
Adressed in 68e462be9645a40c66358a4c2964875abf81b9bf and a05f55d0d796886a1f2b3556327e028905b506cb